Control coverage/Feature/AppService.md

```diff
@@ -361,6 +361,14 @@ Set-AzWebAppSlot -ResourceGroupName <RGName> -Name <WebAppName> -Slot <SlotName>
 - ARM API to list properties of all slots of app service: <br />
 /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Web/sites/{site}/slots?api-version=2019-08-01<br />
 **Properties:**  properties.httpsOnly
+
+- ARM API to list all security assessments in a Subscription:
```

Comment:
assessments in a Subscription -> assessments **for** a Subscription
can we make the same change at all the places